.eslintrc

```diff
@@ -0,0 +1,222 @@
+{
```

Comment:
I'm not changing how the project is linted, I just added an .eslintrc file because my editor is configured with eslint.